containers/ecr-viewer/src/app/api/migrate-db/migrations/core/index.ts

```diff
@@ -2,5 +2,7 @@ export default {
   "19700101000000_initial.ts": require("./19700101000000_initial"),
   "20250425111400_user_prog.ts": require("./20250425111400_user_prog"),
   "20250428110000_condition_ref.ts": require("./20250428110000_condition_ref"),
+  "20250430160000_condition_fk.ts": require("./20250430160000_condition_fk"),
```

Comment:
because 202505 has already merged, can you change the condition fk to be later? also, what's the thinking on two migrations instead of doing this in one go?